web/Controllers/LandlordController.cs

```diff
@@ -67,13 +75,15 @@ public async Task<ActionResult> RegisterPost([FromForm] LandlordProfileModel cre
                                  + $"Email: {createModel.Email}" + "\n"
                                  + $"Phone: {createModel.Phone}" + "\n";
                 _mailService.SendMsg(msgBody);
-
-                return Redirect(Url.Action("MyProfile")!);
+                if (!createModel.Unassigned)
+                    return Redirect(Url.Action("MyProfile")!);
+                //return RedirectToAction("LandlordList", "Admin");

```

Comment:
Delete commented out code

Reply:
Apologies, I pushed to save my work midway through fixing the ones above ^^' I didn't realise it would be re-reviewed without pressing the button.

Reply:
If I turn it into a draft in future, will it retain your comments?

Reply:
Yeah I believe so